### PR TITLE
fixed question count error

### LINKED ISSUE #70: Wrong question count
https://github.com/certificationy/certificationy-cli/issues/25

Start command in certificationy-cli gives question count more than actual questions exist.

```
$ php certificationy.php start --number 1000
Starting a new set of 252 questions (available questions: 425)
$ grep "\-$" vendor/certificationy/*/data/* | wc -l
252
```

### PR DESCRIPTION
**Pull request by mazhuravlev**
resolve #70

### CODE REVIEW

File: Certification/Loader.php [hunk 1]
```diff
@@ -21,62 +21,81 @@
...
+     * Loads questions from yaml path
      *
-     * @param integer $number
-     * @param array   $categories
+     * @param array $categories
      *
-     * @return Set
+     * @return Question[]
      */
-    static public function init($number, array $categories, $path)
```
Comment (mickaelandrieu) [inline Certification/Loader.php]: This function is required and part of public API, we cannot remove it :)

**Comment by mazhuravlev** [comment]
I will fix it.

**Comment by mazhuravlev** [comment]
Returned public methods _Loader#init_ and _Loader#count_.

File: Certification/Loader.php [hunk 2]
```diff
@@ -21,62 +21,108 @@
...
+                $questionData['question'],
+                $questionData['category'],
+                $answers
+            );
+        };
+
+        return array_map($mapQuestion, $data);
+    }
+
+    protected static function createAnswer($value, $correct)
```
Comment (clement-michelet) [inline Certification/Loader.php]: It should be documented with phpDoc.

File: Certification/Loader.php [hunk 3]
```diff
@@ -21,62 +21,108 @@
...
+    public static function getCategories($path)
+    {
+        $categories = array();
+        $files = self::prepareFromYaml(array(), $path);
+        foreach ($files as $file) {
+            $categories[] = $file['category'];
         }
 
-        return new Set($questions);
+        return array_unique($categories);
```
Comment (clement-michelet) [inline Certification/Loader.php]: It could be better to re-index the array by calling `array_values` before returning it.

**Comment by mazhuravlev** [comment]
I have less invasive solution for this issue [here](https://github.com/certificationy/certificationy/pull/72).

**Comment by mickaelandrieu** [comment]
closed in favor of https://github.com/certificationy/certificationy/pull/72

**Pull request closed without merging**